pkg/agent/proxy/proxier.go

```diff
@@ -416,6 +428,9 @@ func (p *proxier) installServices() {
 					needUpdateEndpoints = true
 				}
 			}
+			if p.multiclusterEnabled {
+				mcsLocalService = p.getMCSExportedServiceInfo(serviceCIDRIPv4, endpoint.IP(), svcPortName)
```

Comment:
Is it possible that multiple local Services work as Endpoints in a single multi-cluster Service?

Reply:
Since we already find the updated local Service and its OF group ID, why do we still need to add the endpoint into `allEndpointUpdateList` ?

Reply:
1. No, the service name is unique, each Service will have its own multi-cluster Service.
2. If we didn't include it in allEndpointUpdateList, it will have a problem to update MC Service's Group when Service is no longer exported and Endpoint is removed, because there is no change in the cached installed Endpoint map.

Reply:
Done, refined the installed Endpoint map update logic.